crates/libm-cdylib/src/lib.rs

```diff
@@ -0,0 +1,153 @@
+#![allow(dead_code)]
+#![cfg_attr(not(test), feature(core_intrinsics, lang_items))]
+#![cfg_attr(not(test), no_std)]
+
+#[path = "../../../src/math/mod.rs"]
+mod libm;
+
+#[macro_use]
+mod macros;
+
+#[cfg(test)]
+mod test_utils;
+
+#[cfg(not(test))]
+#[panic_handler]
+fn panic(_info: &core::panic::PanicInfo) -> ! {
+    unsafe { core::intrinsics::abort() }
+}
+
+#[cfg(not(test))]
+#[lang = "eh_personality"]
+extern "C" fn eh_personality() {}
```

Comment:
There has to be a better way that avoids this, but I exhausted all my google-fu trying to find it.

Reply:
This has to do with libcore itself being built with panic=unwind, but why not just link to `std` and have it provide these?

Reply:
Because libstd would pull the system's libm.